### PR TITLE
Update lambda.md

### PR DESCRIPTION
**Pull request by almargit**

### CODE REVIEW

File: learn/functions/lambda.md [hunk 1]
```diff
@@ -3,32 +3,49 @@
...
+```python
+students = [('Alice', 25), ('Bob', 20), ('Charlie', 30)]
+students.sort(key=lambda x: x[1])
+print(students)  # Output: [('Bob', 20), ('Alice', 25), ('Charlie', 30)]
 ```
 
-In the first example, a `lambda` function `multiply` is defined that **takes in two parameters** and returns their product. In the second example, the lambda function **is used as an argument** to the `map()` function to square each element in the list. Lambda functions are often used in conjunction with other functions **to make code more concise and readable.**
+Lambda functions are particularly handy when a small function is needed for a brief period. Their simplicity is beneficial in cases where defining a regular function might be overkill.
+
+Key Points to Remember:
```
Comment (angarsky) [inline learn/functions/lambda.md:42]: If it's a heading - highlight it. If it's just a sentence - don't use Camel Case.

File: learn/functions/lambda.md [hunk 2]
```diff
@@ -3,32 +3,49 @@
...
+```python
+lambda arguments: expression
 ```
 
-### Example 2: Using a lambda function as an argument to the map() function
+Here, `lambda` is the keyword, `arguments` are the function's input parameters, and `expression` is the operation the function performs.
 
+Here are also some examples of declaration and usage:
+
+1. **Basic Addition**
```
Comment (angarsky) [inline learn/functions/lambda.md:20]: Let's convert these items to H3 headings. But don't use bold highlighting `**` inside the heading tags.

**Review by angarsky — CHANGES_REQUESTED** [review]
Good job! But let's highlight several headings.

File: learn/functions/lambda.md (hunk 1 above)
Comment (almargit) [reply learn/functions/lambda.md:42]: corrected

File: learn/functions/lambda.md (hunk 2 above)
Comment (almargit) [reply learn/functions/lambda.md]: done

**Review by angarsky — APPROVED** [review]
Approved.

**Pull request merged**